content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -2,135 +2,131 @@
+  :guilabel:`Discount`, :guilabel:`Formula`, or :guilabel:`Fixed Price`. Depending on the price
+  type, there will be additional configurations on how to apply the pricelist.
 
-To add a specific product and price to a pricelist form, click the :guilabel:`Price Rules` tab, then
-click :guilabel:`Add a line` in the :guilabel:`Products` column. Then, select the desired product
-for which a specific price should be applied.
+  - :guilabel:`Discount`: Enter the percentage to be discounted. A mark-up can be configured by
+    using a negative value in this field.
 
-Next, if necessary, select a product variant under the :guilabel:`Variants` column (e.g. a specific
-product size, color, etc.). If no variants are selected, then this price will apply to all variants
-of the product.
+  - :guilabel:`Formula`: Calculate the pricelist rules based the following configuration:
 
-If a minimum amount of the product must be purchased in order to apply the specific price, enter the
-amount under the :guilabel:`Min. Quantity` column.
+    - :guilabel:`Discount`: Percentage discount to be applied.
+    - :guilabel:`Round off to`: Numerical value to act as round-off multiple, to be applied after
+      discount. The rounding method sets the price so that it is a multiple of the value in this
+      field.
 
-To configure the price of the product for this specific pricelist, enter the desired amount under
-the :guilabel:`Price` column. Then, there is the option to add a :guilabel:`Start Date` and
-:guilabel:`End Date` to the configured product price, if desired.
+.. note::
+   Rounding is applied *after* the discount and *before* the extra fee.
+
+    - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount` and
+      :guilabel:`Round off to` have been applied.
+
+  - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product lines
+    in the quotation form will be updated to this price.
+
+- :guilabel:`Min Qty`: Specify the minimum quantity of selected products for this pricelist to
+  apply.
+- :guilabel:`Validity Period`: Specify the start and end date during which this pricelist can be
+  applied to quotations.
 
-To add another product line, click :guilabel:`Add a line` again, and repeat the process. There is no
-limit to how many products can be added in the :guilabel:`Price Rules` tab of a pricelist form.
+.. example::
+   To formulate a 100% markup (or 2 times the cost of the product), with a $5 minimum margin, set
+   the :guilabel:`Based on` field to :guilabel:`Cost`, the :guilabel:`Discount` to `-100`, and the
+   :guilabel:`Margins` to `5`. This is often seen in retail situations.
 
-For more information, check out the following section: :ref:`Multiple prices per product
-<sales/multiple-prices-per-product>`.
+   .. image:: pricing/formula-markup-cost-example.png
+      :alt: How it looks to formulate a markup cost with 5 dollar minimum margin in Odoo Sales.
```

Comment:
This image (`formula-markup-cost-example.png`) is uncompressed, please compress with pngquant :)